databases/schemas/roomlogs.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE public.roomlogs (
+	type STRING NOT NULL,
+	room STRING NOT NULL,
+	userid STRING NULL,
+	day STRING NOT NULL,
+	month STRING NOT NULL,
+	year STRING NOT NULL,
```

Comment:
These are computed columns. I think Cockroach has "views" or whatever, but really the right way to do this is to hunker down and do `WHERE time > {start} AND time < {end}`